packages/app/src/components/SourceForm.tsx

```diff
@@ -724,31 +731,34 @@ export function TableSourceForm({
             render={({ field: { onChange, value } }) => (
               <Radio.Group
                 value={value}
-                onChange={v => onChange(v as 'log' | 'trace')}
+                onChange={v => onChange(v)}
                 withAsterisk
               >
                 <Group>
-                  <Radio value="log" label="Log" />
-                  <Radio value="trace" label="Trace" />
+                  <Radio value={SourceKind.Log} label="Log" />
+                  <Radio value={SourceKind.Trace} label="Trace" />
+                  {IS_SESSIONS_ENABLED && (
```

Comment:
enabling only for dev

Reply:
is the intent that we'll follow up with a session model/source form later?